src/table/table.scss

```diff
@@ -78,7 +78,7 @@
       list-style: none;
       flex-basis: 145px;
 
-      > .iui-user-icons-small {
+      > .iui-user-icon & .iui-small {
         margin-right: $iui-s;
```

Comment:
Seems like a duplication with below change...
(ideally, we should not have anything in table to handle these specific cases)

Reply:
You are correct, I removed the duplicated code.